index.html

```diff
@@ -0,0 +1,362 @@
+          <img class="client-card__icon" src="img/stars.png" />
+          <h6 class="client-card__description">
+            Slate helps you see how many more days you need to work to reach
+            your financial goal.
+          </h6>
+          <div class="client-card__footer">
+            <img
+              class="client-card__footer-img"
+              src="img/media circle-box xs-circle.png"
+            />
+            <div class="client-card__footer-names">
+              <a class="client-card__footer-name" href="#">Regina Miles</a>
+              <h6 class="client-card__footer-title">Designer</h6>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="general-section">
+      <div class="section">
+        <h2 class="section__title">Meet Our Team</h2>
+        <p class="section__description">
+          Problems trying to resolve the conflict between the two major realms
+          of Classical physics: Newtonian mechanics
+        </p>
+      </div>
+
+      <div class="general-section__cards general-section__cards--big-size">
+        <div class="team-card">
+          <img class="team-card__img" src="img/cover.png" />
+          <div class="team-card__content">
+            <h5 class="team-card__name">Ashley Fletcher</h5>
+            <h6 class="team-card__profession">CEO</h6>
+            <div class="team-card__icons">
+              <img class="team-card__icon" src="img/fb.png" />
+              <img class="team-card__icon" src="img/IG.png" />
+              <img class="team-card__icon" src="img/twitter.png" />
+            </div>
+          </div>
+        </div>
+        <div class="team-card">
+          <img class="team-card__img" src="img/cover.png" />
+          <div class="team-card__content">
+            <h5 class="team-card__name">Ashley Fletcher</h5>
+            <h6 class="team-card__profession">CEO</h6>
+            <div class="team-card__icons">
+              <img class="team-card__icon" src="img/fb.png" />
+              <img class="team-card__icon" src="img/IG.png" />
+              <img class="team-card__icon" src="img/twitter.png" />
+            </div>
+          </div>
+        </div>
+        <div class="team-card">
+          <img class="team-card__img" src="img/cover.png" />
+          <div class="team-card__content">
+            <h5 class="team-card__name">Ashley Fletcher</h5>
+            <h6 class="team-card__profession">CEO</h6>
+            <div class="team-card__icons">
+              <img src="img/fb.png" />
+              <img src="img/IG.png" />
+              <img src="img/twitter.png" />
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="contact-section">
+      <h5 class="contact-section__title">Contact Us</h5>
+      <div class="section">
+        <h2 class="section__title">Make an Appointment</h2>
+      </div>
+
+      <div class="contact-section__form">
+        <div>
```

Comment:
DIV elements with no class. These are unnecessary. If you need them give them a className.